pkgs/development/compilers/bambu/default.nix

```diff
@@ -0,0 +1,237 @@
+{ runCommand
+, makeWrapper
+, buildEnv
+, fetchFromGitHub
+, stdenv
+, lib
+, pkgsi686Linux
+, pugixml
+, libmpc
+, gmp
+, mpfr
+, boost
+, libxml2
+, xz
+, zlib
+, icu
+, suitesparse
+, mpfi
+, glpk
+, autoconf
+, autoconf-archive
+, automake
+, libtool
+, bison
+, doxygen
+, flex
+, graphviz
+, verilog
+, verilator
+, gmpxx
+, buddy
+, bash
+  # 32bit Clang with a supported major version
+, clangFrontends ? [ pkgsi686Linux.llvmPackages_7 pkgsi686Linux.llvmPackages_8 pkgsi686Linux.llvmPackages_10 pkgsi686Linux.llvmPackages_11 pkgsi686Linux.llvmPackages_12 pkgsi686Linux.llvmPackages_13 pkgsi686Linux.llvmPackages_16 ]
```

Comment:
```suggestion
, clangFrontends ? [ pkgsi686Linux.llvmPackages_8 pkgsi686Linux.llvmPackages_10 pkgsi686Linux.llvmPackages_11 pkgsi686Linux.llvmPackages_12 pkgsi686Linux.llvmPackages_13 pkgsi686Linux.llvmPackages_16 ]
```

LLVM 7 is gone.

Reply:
Sorry, I rebased to the latest nixpkgs at the end and but failed to test that properly.